lib/node_modules/@stdlib/math/generics/random/sample/README.md

```diff
@@ -162,19 +164,39 @@ var out = mysample( [1,2,3] );
 // e.g., returns [ 3, 1, 2 ]
 ```
 
-When fixing the population to sample from, the created function returns `null` after all population units are exhausted.
+If a population from which to sample from is provided, the underlying `pool` is reset after each function invocation. To mutate the `pool` by removing observations when sampling *without* replacement permanently, set the `mutate` option.
```

Comment:
Stating that the `pool` is "reset" is somewhat misleading and implies we are doing something under the hood which we are not (i.e., actively managing state and thus memory, etc). Instead,

> If a population from which to sample is provided, the underlying `pool` remains constant for each function invocation.

Also, notice the grammar edit in the dependent clause.